### PR TITLE
Remove some unnecessary 3p deps; Further work on #808

### PR DESCRIPTION
**Pull request by chrisdostert**

### CODE REVIEW

File: sdks/go/data/fs/handle.go [hunk 1]
```diff
@@ -53,7 +48,7 @@ func (lh handle) rListDescendants(
 	[]*model.DirEntry,
 	error,
 ) {
-	childFileInfos, err := lh.ioUtil.ReadDir(path)
+	childFileInfos, err := ioutil.ReadDir(path)
```
Comment (apexskier) [inline sdks/go/data/fs/handle.go:51]: Just an FYI if you hadn't seen, `io/ioutil` is being deprecated in go 1.16. This'll be moved to `os.ReadDir`

**Review by apexskier — APPROVED** [review]
We need to get gofmt PR checks added, there's some violations being added here. I'm not commenting on them explicitly since it's so easily automatable though and is just noise.

**Comment by apexskier** [comment]
> We need to get gofmt PR checks added, there's some violations being added here. I'm not commenting on them explicitly since it's so easily automatable though and is just noise.

https://github.com/opctl/opctl/pull/877

File: sdks/go/data/fs/handle.go (hunk 1 above)
Comment (chrisdostert) [reply sdks/go/data/fs/handle.go:51]: Nice. I gave a quick shot at upgrading to 1.16 but tests weren't passing so whoever gets to it first will have to look into it; assuming won't be too bad. We should ditch `io/ioutil` when we do.

**Comment by chrisdostert** [comment]
> > We need to get gofmt PR checks added, there's some violations being added here. I'm not commenting on them explicitly since it's so easily automatable though and is just noise.
> 
> #877

awesome

**Pull request merged**